report-api/src/main/resources/template/publication.sparql

```diff
@@ -16,44 +16,38 @@ SELECT DISTINCT
     ?publicationIdentifier
 WHERE {
   GRAPH ?g {
-    ?uri a :Publication ;
+    ?uri a ?instanceType ;
          :status ?publicationStatus ;
          :modifiedDate ?modifiedDate ;
          :identifier ?publicationIdentifier ;
-         :entityDescription ?entityDescription .
+         :year ?year ;
+         :mainTitle ?publicationTitle ;
+         :publicationContext ?publicationContext .
 
     FILTER(?modifiedDate >= "__AFTER__"^^xsd:dateTime)
     FILTER(?modifiedDate < "__BEFORE__"^^xsd:dateTime)
 
     BIND(STR(?uri) AS ?publicationId)
+    BIND(REPLACE(STR(?instanceType), "https://nva.sikt.no/ontology/publication#", "", "i") AS ?publicationCategory)
```

Comment:
In a previous query, I notice we still use the complex regex variant…maybe we need to, but check usages of this as it is difficult to grok.